tasks.py

```diff
@@ -383,9 +383,12 @@ def test_addons():
 
 
 @task
-def test(all=False):
+def test(all=False, syntax=False):
```

Comment:
Any reason this should not be `True` by default? From experience I've many times waited optimistically for a travis build to start only to find that my local tests pass but I had forgot to run `flake` before a push.